src/perf/impeller.md

```diff
@@ -74,6 +74,44 @@ include a small reproducible test case.
 
 [file-issue]: {{site.github}}/flutter/flutter/issues/new/choose
 
+### macOS
+
+Impeller is available for macOS in preview as of the 
+Flutter 3.13 stable release.
+
+* To _enable_ Impeller on iOS when debugging,
```

Comment:
Here and elsewhere, there are a couple of instances where the patch refers to iOS :)

Reply:
Whoops thanks!